fastlane/Fastfile

```diff
@@ -349,24 +372,6 @@ platform :ios do
     )
   end
 
-  #####################################################################################
-  # finalize_hotfix_release
-  # -----------------------------------------------------------------------------------
-  # This lane finalizes the hotfix branch.
-  # -----------------------------------------------------------------------------------
-  # Usage:
-  # bundle exec fastlane finalize_hotfix_release [skip_confirm:<skip confirm>]
-  #
-  # Example:
-  # bundle exec fastlane finalize_hotfix_release skip_confirm:true
-  #####################################################################################
-  desc 'Performs the final checks and tags the hotfix in the current branch'
-  lane :finalize_hotfix_release do |options|
-    ios_finalize_prechecks(options)
-    version = ios_get_app_version
-    trigger_release_build(branch_to_build: "release/#{version}")
-  end
-
```

Comment:
moved to lines 253-270 above so it gets closer to the related `new_hotfix_release` lane and is following the same lane order than in other app's Fastfiles for consistency.

Reply:
Good idea. Spatial locality ++